lock-api-objects/src/main/java/com/palantir/lock/watch/NoOpLockWatchEventCache.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.lock.watch;
+
+import java.util.OptionalLong;
+import java.util.Set;
+import java.util.UUID;
+
+import com.google.common.collect.ImmutableSet;
+
+@SuppressWarnings("FinalClass") // mocks
```

Comment:
just make it final and use `mock-maker-inline` which can mock final types: https://www.baeldung.com/mockito-final

Reply:
is this used in prod? or is this just used for tests, haven't really processed where everything is used just yet